README.md

```diff
@@ -0,0 +1,21 @@
+# ssl-refbox
+RoboCup Small Size League Referee Box
+
+
+to install:
+
+1: clone the repro somewhere
```

Comment:
isn't this quite obvious? ;)

Reply:
this is not a complaint nor a judgment, some people that are, for example,  not quite familiar with coding might want to install ssl-refbox to play with it. I made these steps to be fool-proof for someone who has almost 0 experience with coding.